docs/devconsole/README.md

```diff
@@ -0,0 +1,20 @@
+# GitOps Setup on DevConsole
+
+
+1. Setup a GitOps pipeline on your cluster
+    * Using ODO - https://gist.github.com/rohitkrai03/c01a93739dcfad8300d1e4e6450caba6
+    * Using GitOps-CLI - [Docs not available] Contact gitops teams
+2. Install GitOps service operator from Operator hub in all-namespaces (cannot be installed in a specific ns because of an ongoing issue https://coreos.slack.com/archives/CMP95ST2N/p1600695524017600)
```

Comment:
Do customers have access to this slack? If so, we should probably link to instructions on how to join.

Reply:
I am not finding the link.  @chetan-rns Is this issue fixed?

Reply:
The issue still exists. I've created a Jira ticket to track this https://issues.redhat.com/browse/GITOPS-346

Reply:
So, perhaps we should drop the link to slack?

Reply:
I will drop the link to slack.